Source/SIMPLib/Plugin/SIMPLPluginConstants.h

```diff
@@ -0,0 +1,81 @@
+
+
+#pragma once
+
+#include <QtCore/QString>
+
+namespace SIMPL
+{
+namespace JSON 
+{
+
+static const QString ErrorCode("ErrorCode");
```

Comment:
It will take more effort to change this than it's worth, but consider using `extern const` instead of `static const` for namespace constants in the future.  Inside of a namespace, static will create a new SIMPL::JSON::* for each .cpp file this header is included in.  `extern` will require a .cpp file to define the variable, but you will only have one copy of each constant regardless of the number of places this header is included (including recursively through other header files that include it), which seems to be the intent here.

No need to change it now, but keep it in mind for later.